elasticdl/python/common/args.py

```diff
@@ -465,6 +465,14 @@ def parse_master_args(master_args=None):
             "checkpoint_filename_for_init is required for running "
             "prediction job"
         )
+    if not args.use_async and args.get_model_steps > 1:
+        raise ValueError(
+            "get_model_steps should be 1 when using synchronous SGD."
```

Comment:
We can ignore this argument and warn the users instead of crashing our program.

Reply:
Done.